gatsby-config.js

```diff
@@ -15,15 +15,23 @@ module.exports = {
     pages: [
       {
         title: 'Adobe Express Embed SDK',
-        path: '/'
+        path: '/',
       },
       {
         title: 'Guides',
         path: '/guides/'
       },
       {
         title: 'Reference',
-        path: '/reference/'
+        menu: [{
+          title: 'v2.0.1',
+          description: 'v2 Reference', // optional,
+          path: '/v2/reference/'
+        }, {
+         title: 'v1.5.4',
+         description: 'v1 Reference', // optional,
+         path: '/reference/'
```

Comment:
Shall we make it /v1/ reference. That ways it will unambiguous and we can leave the default path with current . /reference symlink or redirct to -> /v2/reference or which ever is current version.

Reply:
@agrawaltarun I committed and pushed new changes according to your feedback here. Please take a look.

Reply:
Looks good, Thanks